### PR TITLE
[Sp24 DS100] 1. Admin access (thru bcourses `Admins` user group) 2. Grant DS100 Content Team 4G RAM.

### PR DESCRIPTION
**Pull request by CharlieJCJ**
Start of semester datahub access request. 

Thanks,
Charlie

### CODE REVIEW

**Comment by balajialg** [comment]
@CharlieJCJ thanks for creating this PR! minor edit - Can you please link the github issue https://github.com/berkeley-dsep-infra/datahub/issues/5376 as a comment in the PR?

**Comment by CharlieJCJ** [comment]
For sure! [Issue](https://github.com/berkeley-dsep-infra/datahub/issues/5376) linked.

**Comment by CharlieJCJ** [comment]
Yeah, just a note. I added a commit to revert back to prev fa23 package version and will open a separate PR, issue for package updates in sp24.

**Comment by CharlieJCJ** [comment]
@balajialg waiting for merge approval, thanks!

**Comment by balajialg** [comment]
@CharlieJCJ Sounds good. Seems like I didn't communicate the change to the PR properly last time. Can you please link the github issue https://github.com/berkeley-dsep-infra/datahub/issues/5376 as a comment in the PR for our internal tracking purposes? ([Example linked](https://github.com/berkeley-dsep-infra/datahub/pull/5366/commits/88b044161806bd8473b18d5e8cdfa2bdfece7214)). Infra admins prefer that approach for all PRs making changes to config and images. 

I will merge the PR to staging ASAP. Having said that, @shaneknapp intends to merge all the changes to prod after 5 PM today. So, these changes will only go live tonight. Hope that works for you and your team @CharlieJCJ

File: deployments/data100/image/environment.yml [hunk 1]
```diff
@@ -49,4 +49,4 @@ dependencies:
   - pytest-notebook==0.8.1
   - gh-scoped-creds==4.1
   - git+https://github.com/shaneknapp/python-popularity-contest.git@add-error-handling
-  - ydata-profiling==4.6.4
+  - ydata-profiling==4.6.4
```
Comment (shaneknapp) [inline deployments/data100/image/environment.yml]: could you revert any changes to this file please?  thanks!

**Review by shaneknapp — CHANGES_REQUESTED** [review]

File: deployments/data100/image/environment.yml (hunk 1 above)
Comment (CharlieJCJ) [reply deployments/data100/image/environment.yml]: Done, thanks for pointing out!

**Comment by CharlieJCJ** [comment]
Fixed all. Thanks!

**Review by ryanlovett — APPROVED** [review]
@balajialg Looks good to me.

**Pull request merged**